src/api/app/models/review.rb

```diff
@@ -60,10 +60,10 @@ class NotFoundError < APIError
   scope :opened, -> { where(state: :new) }
   scope :accepted, -> { where(state: :accepted) }
   scope :declined, -> { where(state: :declined) }
-  scope :for_staging_projects, -> { includes(:project).where.not(projects: { staging_workflow_id: nil }) }
-  scope :for_non_staging_projects, -> { includes(:project).where(projects: { staging_workflow_id: nil }) }
+  scope :for_staging_projects, ->(project) { includes(:project).where(projects: { staging_workflow: Staging::Workflow.find_by(project:) }).where.not(projects: { staging_workflow_id: nil }) }
```

Comment:
The value is missing for the `project:` key in the `Staging::Workflow#find_by(...)` call.